### ISSUE TITLE
about loss

### ISSUE THREAD

**Issue by babyta** [chufengt/iccv19_attribute#19]
老师您好：
       关于损失函数，loss = cur_weights *  (target * torch.log(output + EPS)) + ((1 - target) * torch.log(1 - output + EPS))
      是否因该改为这个形式loss = cur_weights * ( (target * torch.log(output + EPS)) + ((1 - target) * torch.log(1 - output + EPS))),我对比过BCEWithLogitsLoss与binary_cross_entropy_with_logits权重应该是乘以整体交叉熵结果的

**Comment by chufengt** [comment]
same as issue #6

**Comment by babyta** [comment]
i see it ,thank you.
